primitives/core/src/offchain/testing.rs

```diff
@@ -375,6 +376,12 @@ impl offchain::Externalities for TestOffchainExt {
 			Err(HttpError::IoError)
 		}
 	}
+
+	fn get_node_public_key(&mut self) -> Result<NodePublicKey, ()> {
+		Ok(NodePublicKey::Ed25519(ed25519::Public([0u8; 32])))
+	}
+
+	fn set_reserved_nodes(&mut self, _nodes: Vec<NodePublicKey>, _reserved_only: bool) {}
```

Comment:
```suggestion
	fn set_reserved_nodes(&mut self, _nodes: Vec<NodePublicKey>, _reserved_only: bool) { unimplemented!() }
```

Maybe better to panic in tests for now, so that people are aware this was never used yet?